test/Microsoft.Sbom.Api.Tests/SBOMGeneratorTest.cs

```diff
@@ -100,4 +102,25 @@ public async Task When_GenerateSbomAsync_WithNoRecordedErrors_Then_EmptyEntityEr
         mockRecorder.Verify();
         mockWorkflow.Verify();
     }
+
```

Comment:
Code coverage for the generator hits most of the happy paths, but leaves a lot of error cases untouched. Should we try to cover more of the error paths?

Reply:
I cover most of the SPDX 30 generation cases in `\source\repos\sbom-tool\test\Microsoft.Sbom.Parsers.Spdx30SbomParser.Tests\Parser\GeneratorTests.cs`. This file is specifically for api generator tests, which didn't change much because I didn't have to add any api params to the generator (compared to the parser which had an actual api param change). 

If there's a code coverage report or something that's not covered specifically, I can take a look, feel free to lmk what you think